### PR TITLE
[IMP] theme_anelusia, *: adapt `s_key_images`

### PR DESCRIPTION
**Pull request by Brieuc-brd**
*: theme_aviato, theme_beauty, theme_bewise, theme_bistro, theme_bookstore, theme_buzzy, theme_clean, theme_cobalt, theme_enark, theme_graphene, theme_kea, theme_kiddo, theme_monglia, theme_nano, theme_notes, theme_odoo_experts, theme_orchid, theme_paptic, theme_real_estate, theme_treehouse, theme_vehicle, theme_yes, theme_zap

This commit adapts customizations for `s_key_images`, added in https://github.com/odoo/odoo/pull/177463

task-4105059
Part of task-4077427

Requires:
- https://github.com/odoo/odoo/pull/177463

### CODE REVIEW

File: theme_anelusia/views/snippets/s_key_images.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<template id="s_key_images" inherit_id="website.s_key_images">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc5" separator=" "/>
+    </xpath>
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_key_images_default_image_1/web_editor/geometric/geo_square_6.svg</attribute>
+        <attribute name="data-shape">web_editor/geometric/geo_square_6</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">s_key_images_default_image_1.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
```
Comment (chgo-odoo) [inline theme_anelusia/views/snippets/s_key_images.xml:13]: This attribute doesn't seem to be needed, could you check ?

Other occurrences as well

**Review by chgo-odoo — CHANGES_REQUESTED** [review]
Thanks @Brieuc-brd, one small comment for you 👍 

For the rest, LGTM !

File: theme_bewise/views/image_content.xml [hunk 2]
```diff
@@ -222,5 +222,26 @@
...
+    <field name="key">website.s_key_images_default_image_3</field>
+    <field name="name">website.s_key_images_default_image_3</field>
+    <field name="url">/theme_bewise/static/src/img/content/college_media_2.jpg</field>
+</record>
+<record id="s_key_images_default_image_4" model="theme.ir.attachment">
+    <field name="key">website.s_key_images_default_image_4</field>
+    <field name="name">website.s_key_images_default_image_4</field>
+    <field name="url">/theme_bewise/static/src/img/content/college_students.jpg</field>
+</record>
+
```
Comment (chgo-odoo) [inline theme_bewise/views/image_content.xml:245]: Extra space here 😅

File: theme_anelusia/views/snippets/s_key_images.xml (hunk 1 above)
Comment (Brieuc-brd) [reply theme_anelusia/views/snippets/s_key_images.xml:13]: I think it's still necessary, but I'm not 100% sure 🤔. 
Let's see during the tech. review what would be the best solution :) .

**Review by chgo-odoo — APPROVED** [review]
LGTM, thanks @Brieuc-brd 😃

**Review by bvr-odoo — CHANGES_REQUESTED** [review]

File: theme_anelusia/views/images_library.xml [hunk 3]
```diff
@@ -242,5 +242,25 @@
     <field name="name">website.s_accordion_image_default_image</field>
     <field name="url">/theme_anelusia/static/src/img/snippets/s_picture.jpg</field>
 </record>
+<record id="s_key_images_default_image_1" model="theme.ir.attachment">
+    <field name="key">website.s_key_images_default_image_1</field>
+    <field name="name">website.s_key_images_default_image_1</field>
+    <field name="url">/theme_anelusia/static/src/img/snippets/s_key_images_default_image_1.jpg</field>
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_anelusia/views/images_library.xml:248]: It depends on the case, but here I preferred to introduce new images to get a better result, while for other themes the current images were good enough to be used :)

File: theme_paptic/views/customizations.xml [hunk 4]
```diff
@@ -473,4 +473,26 @@
     </xpath>
 </template>
 
+<!-- ==== KEY IMAGES ===== -->
+<template id="s_key_images" inherit_id="website.s_key_images" name="Paptic s_key_images">
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_paptic/views/customizations.xml:477]: Most of the templates implemented in the `customizations.xml` files have a name. I think the templates that don't have name have been forgotten 🙂 . I've double-checked and added names to the missing `s_key_images` templates located in this kind of files.

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Review by bvr-odoo — APPROVED** [review]

**Comment by bvr-odoo** [comment]
@Brieuc-brd still conflicts here :/

**Comment by Brieuc-brd** [comment]
Rebased @bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**